### PR TITLE
[FLOC-2122] Remove fedora20 docs

### PR DESCRIPTION
**Pull request by adamtheturtle**
Fixes https://clusterhq.atlassian.net/browse/FLOC-2122

A tool was created to help with this.
A PR for it is at https://github.com/ClusterHQ/internal-tools/pull/7.

### CODE REVIEW

**Comment by adamtheturtle** [comment]
This does just a bit more than the bare minimum to remove the docs. Follow-up issues are dealing with getting rid of all the supporting code.

File: flocker/provision/_install.py [hunk 1]
```diff
@@ -783,7 +783,7 @@ def task_install_flocker(
             'apt-get', '-y', '--force-yes', 'install', package]))
 
         return sequence(commands)
-    else:
+    elif distribution in ('centos-7',):
```
Comment (jongiddy) [inline flocker/provision/_install.py]: Add an else at the end of this `if` to raise UnsupportedDistribution.

**Comment by itamarst** [comment]
Assuming a follow-up issue will deal with `flocker/provision` mentions, merge forward, merge when buildbot is happy and you've addressed any minor comments above.

**Comment by adamtheturtle** [comment]
> Assuming a follow-up issue will deal with flocker/provision mentions

Yes, it should be ready to submit in a few minutes.

**Comment by adamtheturtle** [comment]
* after this has been merged

**Pull request merged**